### PR TITLE
RHMAP-7826 Use new version of fh-db

### PR DESCRIPTION
**Pull request by aliok**

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -23,9 +23,9 @@
       "resolved": "https://registry.npmjs.org/eyes/-/eyes-0.1.8.tgz"
     },
     "fh-db": {
-      "version": "1.1.0",
-      "from": "fh-db@1.1.0",
-      "resolved": "https://registry.npmjs.org/fh-db/-/fh-db-1.1.0.tgz",
+      "version": "1.1.1-114",
+      "from": "fh-db@1.1.1-114",
+      "resolved": "http://npm.skunkhenry.com/fh-db/-/fh-db-1.1.1-114.tgz",
```
Comment (wei-lee) [inline npm-shrinkwrap.json]: if the change in fh-db is ok, I say we publish it and re-generate the shrinkwrap file. This is ok for testing, but we don't want to release fh-mbaas-api with this.

Comment (pb82) [inline npm-shrinkwrap.json]: Yes that was the idea. We would merge the PR only after changing this to a non-development version.

**Comment by pb82** [comment]
LGTM

**Pull request merged**